mttex.sty

```diff
@@ -79,12 +79,20 @@
 \DeclareOption{techrpt}{\secc@techrpttrue}
 \DeclareOption{paper}{\secc@techrptfalse}
 
+%hyperref declaration?
```

Comment:
Sorry, only unnecessary if the reason for this was to use better hyperref options.